sample-app/Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 
 # Our gems
 # bundle config set local.aws-sdk-rails ../
-gem 'aws-sdk-rails', git: 'https://github.com/aws/aws-sdk-rails', branch: 'main'
+gem 'aws-sdk-rails', path: "../"
```

Comment:
Don't forget to change this back - you can set the branch and use bundler config local.